src/css/tickets.css

```diff
@@ -0,0 +1,120 @@
+/*Tickets page CSS*/
+
+
+ /* announcements bar */
+.announcements-container {
+  display: flex;
+  align-items: center;
+  background-color: rgb(231, 211, 31);
+  padding-top: 5px;
+  padding-bottom: 5px;
+}
+
+.announcements-container > div {
+  background-color: rgb(231, 211, 31);
+  color: rgb(0, 0, 0);
+  margin: 0px;
+  text-align: left;
+  line-height: 25px;
+  font-size: 20px;
+  padding: 0px 60px 0px 60px;
+}
+
+  
+ /* ticket information */ 
+.ticket-container {
+    width: 30%;
+    padding: 15px 25px 35px 0px; /*size*/
+}
+
+.ticket-label {
+    border: none;
+    color: white;
+    padding: 15px 60px;
+    text-align: left;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+    margin: 4px 2px;
+	background-color: #bc3434;
+}
+
+.price{
+  border: none;
+  color: white;
+  text-align: right;
+  text-decoration: none;
+  font-size: 18px;
+  margin: 4px -50px;
+}
+
+
+ /* hero image and text */
+.hero-image {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url("../public/images/img2.jpg");
+  height: 50%;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.hero-text {
+	position: absolute;
+    color: #ffff;
+    font-size: 300%;
+    background-color: #359953;
+    padding-top: 15px;
+    padding-bottom: 15px;
+    padding-left: 15px;
+    padding-right: 15px;
+    top: 35%;
+    left: 5%;
+}
+
+ /* payment and contact information */
+.payment-contact-container {
+  display: flex;
+  align-items: stretch;
+  background-color: DodgerBlue;
+  width: 100%;
+  padding: 50px 50px 30px 20px;
+  
+}
+
+.payment-contact-container > div {
+  background-color: rgb(231, 211, 31);
+  color: rgb(2, 2, 2);
+  border: 4px black;
+  margin: 30px;
+  text-align: center;
+  line-height: 45px;
+  font-size: 30px;
+  border: 2px solid rgb(0, 0, 0);
+}
+
+ /* animated mascot */
+.mascot{
+  background-position: center;
+  padding: 30px 30px 0px 100px;
+
+}
+
+ /* park map */
+.map-container {
+  padding: 50px 50px 30px 20px;
+  background-color: rgb(231, 211, 31);
+  color: rgb(2, 2, 2);
+  border: 4px black;
+  margin: 30px;
+  font-size: 30px;
+  border: 2px solid rgb(0, 0, 0);
+  width: 80%;
+  height: 80%;
+}
+
+.map-image {
```

Comment:
Try to remove this class possibly? Seems like `.map-container` should already be enough for the children elements.